docs/notes/bugfix-21769.md

```diff
@@ -0,0 +1 @@
+# Added new function iphoneDeviceModel() that returns the full name of the iOS device the app is running on
```

Comment:
@livecodepanos This is inaccurate as the function no longer maps the machine name to a readable full device name